<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>301</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="1">
     <widget class="QWidget" name="widgetWeight" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="AlexRadioButton" name="radioButton_A">
         <property name="text">
          <string>Arial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AlexRadioButton" name="radioButton_TNR">
         <property name="text">
          <string>Times New Roman</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="AlexRadioButton" name="radioButton_CS">
         <property name="text">
          <string>Comic Sans</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Font Size</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Font Size Type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="AlexLineEdit" name="lineEditFamily"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Family</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="AlexComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>small</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>medium</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>big</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelFox">
      <property name="text">
       <string>The quick brown fox likes CSE 335</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelFamily">
      <property name="text">
       <string>Font Size Type</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="HorizontalSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>labelFamily</zorder>
   <zorder>lineEditFamily</zorder>
   <zorder>label</zorder>
   <zorder>widgetWeight</zorder>
   <zorder>label_3</zorder>
   <zorder>labelFox</zorder>
   <zorder>label_2</zorder>
   <zorder>comboBox</zorder>
   <zorder>horizontalSlider</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AlexLineEdit</class>
   <extends>QLineEdit</extends>
   <header>alexlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>AlexComboBox</class>
   <extends>QComboBox</extends>
   <header>alexcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>AlexRadioButton</class>
   <extends>QRadioButton</extends>
   <header>alexradiobutton.h</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSlider</class>
   <extends>QSlider</extends>
   <header location="global">horizontalslider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
